src/components/withLocalize.js

```diff
@@ -11,14 +11,12 @@ import numberFormat from '../libs/numberFormat';
 
 const withLocalizePropTypes = {
     // Translations functions using current User's preferred locale
-    translations: PropTypes.shape({
-        translate: PropTypes.func.isRequired,
-        numberFormat: PropTypes.func.isRequired,
-        timestampToRelative: PropTypes.func.isRequired,
-        timestampToDateTime: PropTypes.func.isRequired,
-        toLocalPhone: PropTypes.func.isRequired,
-        fromLocalPhone: PropTypes.func.isRequired,
-    }),
+    translate: PropTypes.func.isRequired,
```

Comment:
I know you didn't write this component, but each of these props should have a comment description